src/components/pageComponent/login/loginForm.tsx

```diff
@@ -0,0 +1,107 @@
+import {ChangeEvent, FormEvent, useEffect, useState} from "react";
+import {useRouter} from "next/router";
+import {useSelector} from "react-redux";
+import {RootState} from "@/src/redux/index";
+
+import style from "./LoginForm.module.scss";
+import Input from "@/src/components/common/input/input";
+import Label from "@/src/components/common/input/label";
+import ButtonCta from "@/src/components/common/buttonCta/buttonCta";
+import {saveLoginInfo} from "@/src/data/data-login";
+import {loginContainerErrorMessage} from "@/src/utils/errorMessage";
+
+interface values {
```

Comment:
타입은 파스칼케이스로 작성하는게 컨벤션입니다~
추가적으로 어떤 값인지 알 수 없는 네이밍이네요~
```suggestion
interface Values {
```